content/applications/websites/ecommerce/managing_products/variants.rst

```diff
@@ -1,110 +1,88 @@
+   customers are unable to select the excluded combination of values. To do so, access the
+   :ref:`Configure <variants-configurator-backend>` menu, select a **value**, and fill in the
+   :guilabel:`Exclude for` line.
 
-Set specific prices per variant
-===============================
+Product specifications
+======================
 
-You can set a specific public price per variant by clicking on *Configure Variants* in the product
-detail form (top-left corner). That takes you to the *Product Variant Values* page for that product.
+Values used for each attribute are displayed as a **specification list** at the bottom of the
+product page. To be visible, the **specification list** must first be enabled on the product page by
+going to :menuselection:`Edit --> Customize` and selecting a placement for the field
+:guilabel:`Specification`.
 
-.. image:: variants/configure-variants.png
+.. image:: variants/variants-specifications.png
    :align: center
-   :alt: ecommerce product variants configure variants
-
-To set a specific public price for a particular variant, simply click on the variant you want to
-modify, then click on *Edit*.
+   :alt: Specifications list on the product page
 
-The *Value Price Extra* field represents the monetary value that is added to the original product
-price whenever the corresponding attribute value is selected.
+.. tip::
+   The product **specification list** can also be used on products without variants. For that, make
+   sure to have no values combination. Products with single values for their attributes do not
+   generate variants.
 
-.. image:: variants/value-price-extra.png
-   :align: center
-   :alt: ecommerce value price extra
-
-This is how the *Value Price Extra* looks on your website:
-
-.. image:: variants/value-price-extra-frontend.png
-   :align: center
-   :alt: ecommerce value price extra on the frontend
-
-.. note::
-   Pricelist formulas let you set advanced price computation methods for product variants. See
-   :doc:`../maximizing_revenue/pricing`.
+Filter catalog by attributes
+============================
 
-Disable/archive variants
-========================
+Customers **can** filter the **catalog** based on product attributes and values, allowing them to
+filter the catalog based only on the attribute(s) of their choice.
 
-You can disable/archive specific variants so they are no longer available in quotes or on your
-website (not existing in your stock, deprecated, etc.). Simply select *Archive* in their detail
-form. You can reactivate them the same way.
+To enable **attributes filtering**, go to :menuselection:`Edit --> Customize` from your **main shop
+page** and click on one of the **categories** in the left column. Here, enable either
+:guilabel:`Left`, :guilabel:`Top`, or **both**, in the :guilabel:`Attributes` field.
 
-.. image:: variants/archive.png
+.. image:: variants/variants-catalog.png
```

Comment:
```suggestion
.. image:: variants/variants-catalog.png
```
Not really useful IMO - you don't really know what you are looking at.
If you really want a screenshot here, maybe show the categories option which is a bit difficult to find.
![image](https://user-images.githubusercontent.com/70884563/212948586-8a823938-1c5f-4946-97e3-c125a34c43b6.png)